volatile-requirements.txt

```diff
@@ -2,3 +2,4 @@
 # a lot and usually we need master
 git+https://github.com/esheldon/ngmix.git@master
 git+https://github.com/esheldon/meds.git@master
```

Comment:
We should pull these out of conda.

Reply:
Yep.  I put it there so the tests could run; we can keep this PR open until sxdes gets on conda and then update this PR

Reply:
this is done.